### ISSUE TITLE
Doesn't work in Android  10

### ISSUE THREAD

**Issue by pranishres** [VRGsoftUA/VideoCrop#11]

**Comment by VivekThummar52** [comment]
Please describe a little more information on what's the problem?

**Comment by aldrek** [comment]
I have tested it and it's not working on devices that have Android 10 and 11

**Comment by VivekThummar52** [comment]
I don't think so, because i have used this and it works fine in all the tested devices with Android 10.
Can you please specify where the issue occurs (while opening video or at the time of Saving video)?

**Comment by aldrek** [comment]
As fo From 2 November 2020, app updates must target Android 10 (API level 29) or higher. the target SDK and compile SDK is less than 29 in this repo. when you higher the SDK the error pops up. 
I have found a solution I could make a pr to fix this repo

**Comment by VivekThummar52** [comment]
So, basically the solution is to simply add "requestLegacyExternalStorage=true" in manifest file?

**Comment by aldrek** [comment]
even with requestLegacyExternalStorage still, there is a crash happening 
![Capture](https://user-images.githubusercontent.com/10108221/106284324-c315ff00-624b-11eb-8e4c-284b591ee6fc.PNG)

**Comment by VivekThummar52** [comment]
@aldrek in first line, you can see error "permission denied".
So,can you please check if you have added any required permissions or not?

**Comment by aldrek** [comment]
it's not related to any manifest permission. 
As from android 10, no external binaries are allowed to be executed for security concern
I would recommend using a well-supported library for decoding the video. like this one 
https://github.com/tanersener/mobile-ffmpeg

**Comment by VivekThummar52** [comment]
@aldrek yeah, sorry for that, that's what I'm using since more than one year but i thought you are cropping video using UI based tools,by the way best of luck!!

**Comment by proyo-code** [comment]
you need to update the ffmpeg
